### PR TITLE
Updated citations

### PR DESCRIPTION
**Pull request by NoorhanAbbas**
Updated citations in stat of the Art section ref #6

### CODE REVIEW

File: edubots_surveys.Rnw [hunk 1]
```diff
@@ -294,27 +294,21 @@ The rest of the paper is organized as follows. Next is an overview of what curre
 \section{State of the art}
 \label{sec:soa}
 
-It is essential, when implementing a educational chatbot strategy, to asses the needs of the community prior to implementation \cite{tsivitanidouusers}.
+The Use of Mobile Instant Messaging (MIM) Platforms in Higher Education
```
Comment (JJ) [inline edubots_surveys.Rnw:297]: Is this like a subsection header?

**Comment by JJ** [comment]
@NoorhanAbbas as you see, there's what's called a conflict. This usually happens when you're working on an outdated version of the files. Did you remember to "update" via the button before editing? Can you please check that? Anyway, I think I can solve the issue... ish. So if you can't do it, I can try and fix it from my side.

File: edubots_surveys.Rnw (hunk 1 above)
Comment (NoorhanAbbas) [reply edubots_surveys.Rnw:297]: Yes it is.

**Comment by NoorhanAbbas** [comment]
Hello JJ,
I have updated all the citations in the State of the art section.  Could you please have a quick look and let me know if it looks right now.

Thank you very much,
Noorhan
________________________________

**Comment by JJ** [comment]
You have closed this, I'm afraid... If you want, I can re-open it an deal with its content.

**Comment by NoorhanAbbas** [comment]
Yes please. Thank you very much for your support and patience!

Noorhan
________________________________

**Comment by JJ** [comment]
Done. Please remember to always "Fetch and Merge" (a button on top of the code panel) before submitting any changes.

**Pull request closed without merging**